redfish-core/lib/log_services.hpp

```diff
@@ -802,8 +802,8 @@ inline void deleteDumpEntry(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                 return;
             }
 
-            BMCWEB_LOG_ERROR << "Dump (DBus) doDelete respHandler got error "
-                             << ec << " entryID=" << entryID;
+            BMCWEB_LOG_ERROR << "Dump (DBus) doDelete respHandler got "
+                             << "error: " << ec << "for entryID:" << entryID;
```

Comment:
Single string and add a blank like this?
```
            BMCWEB_LOG_ERROR << "Dump (DBus) doDelete respHandler got error: " << ec << " for entryID: " << entryID;
```

Then, `clang-format-16` will format it like
```
BMCWEB_LOG_ERROR << "Dump (DBus) doDelete respHandler got error: " << ec
                 << " for entryID: " << entryID;
```

Reply:
done